tests/backup/backup_helper.go

```diff
@@ -1187,8 +1201,7 @@ func backupSuccessCheck(backupName string, orgID string, retryDuration int, retr
 	if err != nil {
 		return false, err
 	}
-	backupStatus := (resp.GetBackup().GetStatus().Status == api.BackupInfo_StatusInfo_Success)
-	log.Infof("Backup [%s] created successfully", backupName)
+	backupStatus := resp.GetBackup().GetStatus().Status == api.BackupInfo_StatusInfo_Success
```

Comment:
What @nikolaypopov is mentioning here is, we don't even need this line here. We can remove this line and just return true, nil because this check of resp.GetBackup().GetStatus().Status == api.BackupInfo_StatusInfo_Success is already happening inside the anonymous function above inside task.DoRetryWithTimeout

Reply:
done